src/partitioned-dbs/index.rst

```diff
@@ -0,0 +1,384 @@
+        ]
+    }
+
+
+.. note::
+    While this example uses IoT sensors, the main thing to consider is that
+    there is a logical grouping of documents. Similar use cases might be
+    documents grouped by user or scientific data grouped by experiment.
+
+
+So we've got a bunch of sensors, all grouped by the field they monitor
+along with their readouts for a given day (or other appropriate time period).
+
+Along with our documents we might expect to have two secondary indexes
+for querying our database that might look something like:
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf("sensor-reading-") != 0) {
+            return;
+        }
+        for(var r in doc.readings) {
+            emit([doc.sensor_id, r[0]], r[1])
+        }
+    }
+
+and:
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf("sensor-reading-") != 0) {
+            return;
+        }
+        emit(doc.field_name, doc.sensor_id)
+    }
+
+With these two indexes defined we can easily find all requests for a given
+sensor, or list all sensors in a given field.
+
+Unfortunately, in CouchDB, when we read from either of these indexes, it
+requires finding a copy of every shard and asking for any documents related
+to the particular sensor or field. This means that as our database scales
+up the number of shards, every index request must perform more work.
+Fortunately for you, dear reader, partitioned databases were created to solve
+this precise problem.
+
+
+What is a partition?
+====================
+
+In the previous section, we introduced a hypothetical database that contains
+sensor readings from an IoT field monitoring service. In this particular
+use case, it's quite logical to group all documents by their ``sensor_id``
+field. In this case, we would call the ``sensor_id`` the partition.
+
+A good partition has two basic properties. First, it should have a high
+cardinality. That is, there is a large number of values for the partition.
+A database that has a single partition would be an anti-pattern for this
+feature. Secondly, the amount of data per partition should be "small". The
+general recommendation is to limit individual partitions to less than ten
+gigabytes of data. Which, for the example of sensor documents, equates to roughly
+60,000 years of data.
+
+
+Why use partitions?
+===================
+
+The primary benefit of using partitioned databases is for the performance
+of partitioned queries. Large databases with lots of documents often
+have a similar pattern where there are groups of related documents that
+are queried together often.
+
+By using partitions, we can execute queries against these individual groups
+of documents more efficiently by placing the entire group within a specific
+shard on disk. Thus, the view engine only has to consult one copy of the
+given shard range when executing a query instead of executing
+the query across all ``Q`` shards in the database.
```

Comment:
lowercase q to match the rest of our documentation standards please

Also, might want to say that this means we don't have to wait for all q shards to respond or something, meaning the query is not just more efficient, but faster? Not sure.

Reply:
Addressed in 347a6ce.